src/changelog-generator.js

```diff
@@ -174,7 +174,16 @@ function encodedStr(rawStr)
 function formatChangelogEntry(issue, repository, is_main_repository)
 {
     var pr_link = (is_main_repository ? "" : repository) + '#' + issue.number
-    var description = '  * ' + encodedStr(issue.title) + ' (' + pr_link + ')'
+    var pr_title = encodedStr(issue.title)
+    var jira_regex = /\[(TG-\d+)\](.*)/g;
```

Comment:
I'd change TG for `(TG|SEC)` or something like `[A-Z]{2,4}` to match other Jira projects.

This isn't matching the whole string - you will want a `^` at the start in case the PR is something like "Foo [TG-123] and bar [TG-456]"

Personally, I would use string.replace with a callback as follows:

```typescript
pr_title = pr_title.replace(/\b[A-Z]{2,4}-[0-9]{1,6}\b/g, (match) => `[${match}](https://diffblue.atlassian.net/browse/${match})`);
```